### PR TITLE
PR_AddigPowerSupport_kaniko

### PR DESCRIPTION
**Pull request by santosh653**
creating kaniko.travis.yml with proposed changes.

### CODE REVIEW

File: travis-ymls/kaniko.travis.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+# Travis Job Link     : https://travis-ci.com/github/santosh653/kaniko
+# Created travis.yml  : No
+# Maintainer          : Santosh Kulkarni <santoshkulkarni70@gmail.com>
+#
+# Script License      : Apache License, Version 2 or later
+#
+# ----------------------------------------------------------------------------
+arch:
+  - amd64
+  - ppc64le
```
Comment (Siddhesh-Ghadi) [inline travis-ymls/kaniko.travis.yml:14]: This changes will not create builds properly as build matrix is defined using `jobs`.

File: travis-ymls/kaniko.travis.yml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+dist: bionic
+
+env:
+  global:
+    - IMAGE_REPO=localhost:5000 REGISTRY=localhost:5000
+go:
+  - "1.14"
+go_import_path: github.com/GoogleContainerTools/kaniko
+jobs:
+  include:
```
Comment (Siddhesh-Ghadi) [inline travis-ymls/kaniko.travis.yml:26]: Please add ppc64le jobs under this.

**Review by Siddhesh-Ghadi — CHANGES_REQUESTED** [review]

**Comment by gururajrkatti** [comment]
@santosh653 any updates on this ?

**Comment by gururajrkatti** [comment]
@santosh653 any updates on this ?

**Comment by gururajrkatti** [comment]
@santosh653 any updates on this

**Comment by gururajrkatti** [comment]
@santosh653 any updates on this

**Pull request closed without merging**